qemu/tests/cfg/chardev_tls_encryption.cfg

```diff
@@ -8,6 +8,11 @@
     depends_pkgs = "gnutls-utils"
     ppc64, ppc64le:
         serial_device = "spapr-vty"
+    aarch64:
+        serials += " vs"
+        serial_type_vs = virtserialport
+        device_type = "virtio-serial-pci"
```

Comment:
Oh, the virtserialport needs the device bus to insert it.
```
name "virtio-serial-device", bus virtio-bus
name "virtio-serial-pci", bus PCI, alias "virtio-serial"
```

Reply:
When the serial type is `virtserialport`, the parent bus will be automatically generated.

However, I think you want to add another serial port is because the `extra_params` does not include the virtio-serial-pci, some hard code here, it's better to make VT support tls props if possible.

Reply:
When the serial type is `virtserialport`, the parent bus will be automatically generated.

However, I think you want to add another serial port is because the `extra_params` does not include the virtio-serial-pci, some hard code here, it's better to make VT support tls props if possible.

Reply:
Yes, because `extra_params` contains some hard codes, so I can only add a device for ARM. In view of the fact that the hard code of x86 has been merged, I have no plan to improve it in vt.

Reply:
As I mentioned above, without `device_type = "virtio-serial-pci"`, the QEMU command line still creates the chardev device with the correct parent bus. Have I missed anything?

Reply:
sorry for the delayed update
corrected